pkg/tables/tables.go

```diff
@@ -97,6 +97,20 @@ var (
 	}
 	BuilderConverter = MustConverter(Builder)
 
+	WorkloadClass = [][]string{
+		{"Name", "Name"},
+		{"Default", "{{ boolToStar .Default }}"},
+		{"Description", "Description"},
+	}
+	WorkloadClassConverter = MustConverter(WorkloadClass)
+
+	WorkloadClassInstance = [][]string{
```

Comment:
The main reason they exist like this right now is because of `Spec` existing on the Cluster and Project WorkloadClasses but not the aggregate WorkloadClass